test/uk/gov/hmrc/cdsimportsddsfrontend/controllers/DeclarationController_goodsIdentificationSpec.scala

```diff
@@ -66,7 +73,15 @@ with Scenarios with FutureAwaits with BeforeAndAfterEach {
         "goodsIdentification.description" -> Seq("Our test description"),
         "goodsIdentification.typeOfPackages" -> Seq("boxes"),
         "goodsIdentification.numberOfPackages" -> Seq("13"),
-        "goodsIdentification.shippingMarks" -> Seq("crosses")
+        "goodsIdentification.shippingMarks" -> Seq("crosses"),
+        "goodsIdentification.combinedNomenclatureCode.id" -> Seq("180"),
```

Comment:
can we use different values for each?  otherwise it's possible for the fields to be incorrectly wired up but the test still passes.

Reply:
Agree. Will vary.

Reply:
Agree. Will vary.